x-pack/plugins/apm/server/lib/services/get_service_transaction_group_detailed_statistics.ts

```diff
@@ -68,7 +68,12 @@ export async function getServiceTransactionGroupDetailedStatistics({
   }>
 > {
   const { apmEventClient } = setup;
-  const { intervalString } = getBucketSize({ start, end, numBuckets });
+  const { intervalString } = getBucketSize({
+    start,
+    end,
+    numBuckets,
+    searchAggregatedTransactions,
+  });
```

Comment:
Per my earlier comment I think you should change `searchAggregatedTransactions` to `minBucketSize`:

```suggestion
  const { intervalString } = getBucketSizeForAggregatedTransactions({
    start,
    end,
    numBuckets,
    searchAggregatedTransactions,
  });
```


The implementation for `getBucketSizeForAggregatedTransactions is simply:
```ts
function getBucketSizeForAggregatedTransactions({ 
	start, 
	end, 
	numBuckets, 
	searchAggregatedTransactions 
}: {...}) {
	if (searchAggregatedTransactions) {
		return getBucketSize({ start, end, numBuckets, minBucketSize: 60})
	}

	return getBucketSize({ start, end, numBuckets })
}
```

The implementation for `getBucketSize` changes to:
```ts
export function getBucketSize({
  start,
  end,
  numBuckets = 100,
  minBucketSize
}: {
  start: number;
  end: number;
  numBuckets?: number;
  minBucketSize?: number
}) {
  const duration = moment.duration(end - start, 'ms');
  const bucketSize = Math.max(
    calculateAuto.near(numBuckets, duration).asSeconds(),
    minBucketSize ?? 1,
  );

  return { bucketSize, intervalString: `${bucketSize}s` };
}

```